test fail looks real:
```
======================================================================
FAIL: testRegistrationXML (casexml.apps.phone.tests.test_ota_restore.OtaRestoreTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore.py", line 50, in testRegistrationXML
    xml.get_registration_xml(dummy_user()))
  File "corehq/ex-submodules/casexml/apps/case/tests/util.py", line 103, in check_xml_line_by_line
    "Expected: \n%s\nActual:\n%s" % (parsed_expected, parsed_actual))
AssertionError: 10 != 11 : Parsed xml files are different lengths
Expected: 
<Registration xmlns="http://openrosa.org/user/registration">
  <username>mclovin</username>
  <password>changeme</password>
  <uuid>foo</uuid>
  <date>2011-06-09</date>
  <user_data>
    <data key="something">arbitrary</data>
  </user_data>
</Registration>
Actual:
<Registration xmlns="http://openrosa.org/user/registration">
  <username>mclovin</username>
  <password>changeme</password>
  <uuid>foo</uuid>
  <date>2011-06-09</date>
  <user_data>
    <data key="something">arbitrary</data>
    <data key="commcare_full_name"/>
  </user_data>
</Registration>
----------------------------------------------------------------------
```

--- a/corehq/apps/users/models.py
+++ b/corehq/apps/users/models.py
@@ -1527,7 +1527,7 @@ class CommCareUser(CouchUser, SingleMembershipMixin, CommCareMobileContactMixin)
             loadtest_factor=self.loadtest_factor,
             first_name=self.first_name,
             last_name=self.last_name,
-            phone_numbers=self.phone_numbers,
+            phone_number=self.phone_number,
         )
 
         def get_owner_ids():

--- a/corehq/apps/users/models.py
+++ b/corehq/apps/users/models.py
@@ -1526,7 +1526,7 @@ class CommCareUser(CouchUser, SingleMembershipMixin, CommCareMobileContactMixin)
             domain=self.domain,
             loadtest_factor=self.loadtest_factor,
             full_name=self.full_name,
-            phone_number=self.phone_number,
+            phone_numbers=self.phone_numbers,
         )
 
         def get_owner_ids():

--- a/corehq/ex-submodules/casexml/apps/phone/models.py
+++ b/corehq/ex-submodules/casexml/apps/phone/models.py
@@ -18,18 +18,18 @@ logger = logging.getLogger('phone.models')
 
 
 class User(object):
-    """ 
+    """
     This is a basic user model that's used for OTA restore to properly
     find cases and generate the user XML.
     """
-    
+
     def __init__(self, user_id, username, password, date_joined, full_name=None,
-                 phone_number=None, user_data=None, additional_owner_ids=None,
+                 phone_numbers=None, user_data=None, additional_owner_ids=None,
                  domain=None, loadtest_factor=1):
         self.user_id = user_id
         self.username = username
         self.full_name = full_name
-        self.phone_number = phone_number
+        self.phone_numbers = phone_numbers or []
         self.password = password
         self.date_joined = date_joined
         self.user_data = user_data or {}
@@ -47,7 +47,7 @@ class User(object):
         return cls(user_id=str(django_user.pk), username=django_user.username,
                    password=django_user.password, date_joined=django_user.date_joined,
                    user_data={})
-    
+
 
 class CaseState(LooselyEqualDocumentSchema, IndexHoldingMixIn):
     """

--- a/corehq/ex-submodules/casexml/apps/phone/models.py
+++ b/corehq/ex-submodules/casexml/apps/phone/models.py
@@ -24,13 +24,13 @@ class User(object):
     """
 
     def __init__(self, user_id, username, password, date_joined, first_name=None,
-                 last_name=None, phone_numbers=None, user_data=None,
+                 last_name=None, phone_number=None, user_data=None,
                  additional_owner_ids=None, domain=None, loadtest_factor=1):
         self.user_id = user_id
         self.username = username
         self.first_name = first_name
         self.last_name = last_name
-        self.phone_numbers = phone_numbers or []
+        self.phone_number = phone_number
         self.password = password
         self.date_joined = date_joined
         self.user_data = user_data or {}

--- a/corehq/ex-submodules/casexml/apps/phone/xml.py
+++ b/corehq/ex-submodules/casexml/apps/phone/xml.py
@@ -95,7 +95,9 @@ def get_registration_element(user):
     root.append(safe_element("date", date_to_xml_string(user.date_joined)))
     user_data = user.user_data
     user_data["{}_full_name".format(SYSTEM_PREFIX)] = user.full_name
-    user_data["{}_phone_number".format(SYSTEM_PREFIX)] = user.phone_number
+    for i, phone_number in enumerate(user.phone_numbers):
+        data_key = "{}_phone_number_{}".format(SYSTEM_PREFIX, i+1)
+        user_data[data_key] = phone_number
     root.append(get_data_element('user_data', user.user_data))
     return root
 

--- a/corehq/ex-submodules/casexml/apps/phone/xml.py
+++ b/corehq/ex-submodules/casexml/apps/phone/xml.py
@@ -86,16 +86,17 @@ def get_case_xml(case, updates, version=V1):
     
 
 def get_registration_element(user):
+    from corehq.apps.custom_data_fields.models import SYSTEM_PREFIX
     root = safe_element("Registration")
     root.attrib = {"xmlns": USER_REGISTRATION_XMLNS}
     root.append(safe_element("username", user.username))
-    root.append(safe_element("full_name", user.full_name))
-    root.append(safe_element("phone_number", user.phone_number))
     root.append(safe_element("password", user.password))
     root.append(safe_element("uuid", user.user_id))
     root.append(safe_element("date", date_to_xml_string(user.date_joined)))
-    if user.user_data:
-        root.append(get_data_element('user_data', user.user_data))
+    user_data = user.user_data
+    user_data["{}_full_name".format(SYSTEM_PREFIX)] = user.full_name
+    user_data["{}_phone_number".format(SYSTEM_PREFIX)] = user.phone_number
+    root.append(get_data_element('user_data', user.user_data))
     return root
 
 

--- a/corehq/ex-submodules/casexml/apps/phone/xml.py
+++ b/corehq/ex-submodules/casexml/apps/phone/xml.py
@@ -96,9 +96,7 @@ def get_registration_element(user):
     user_data = user.user_data
     user_data["{}_first_name".format(SYSTEM_PREFIX)] = user.first_name
     user_data["{}_last_name".format(SYSTEM_PREFIX)] = user.last_name
-    for i, phone_number in enumerate(user.phone_numbers):
-        data_key = "{}_phone_number_{}".format(SYSTEM_PREFIX, i+1)
-        user_data[data_key] = phone_number
+    user_data["{}_phone_number".format(SYSTEM_PREFIX)] = user.phone_number
     root.append(get_data_element('user_data', user.user_data))
     return root
 

